adhocracy-plus/assets/scss/components/_project_tile_carousel.scss

```diff
@@ -24,6 +24,7 @@
 
     .slick-track {
         padding-bottom: $padding;
+        margin-left: 0;
```

Comment:
that makes it align left (only affects if we have less projects than 3), otherwise it looks a bit off